tests/SeedTests.elm

```diff
@@ -11,14 +11,19 @@ fixedSeed =
     Random.initialSeed 133742
 
 
+{- This is the expected first int output when running this file with a constant seed.
+
+    (\intFromSeed -> fnvHashString fnvInit "Seed test" |> fnvHash intFromSeed)
+
+-}
 expectedNum : Int
 expectedNum =
-    2800615587
+    -3954212174
```

Comment:
I'm not quite happy with these constants. How did you arrive at this number, @rtfeldman? 

I mean, they're fine for regression testing, and it probably doesn't really matter here, but it just looks so arbitrary :)